docs/content/ingestion/index.md

```diff
@@ -153,6 +153,7 @@ Use this with the String Parser to load CSV. Strings are parsed using the net.sf
 | timestampSpec | JSON Object | Specifies the column and format of the timestamp. | yes |
 | dimensionsSpec | JSON Object | Specifies the dimensions of the data. | yes |
 | listDelimiter | String | A custom delimiter for multi-value dimensions. | no (default == ctrl+A) |
+| multiValueDelimiter | JSON Object | A custom delimiter for multi-value dimensions. | no (default == null) |
```

Comment:
Same as above

Reply:
Same as above, at least add reference to the place where it is explained (multi-value-dimensions.md), and explain what does "no (null)" mean, right here

Reply:
fixed, detail reference to another file